auth/README.md

```diff
@@ -57,6 +57,11 @@ repositories {
 }
 ```
 
+#### Important note
+
+Firebase-UI-auth v1.0.0 is incompatible with the v25 support libraries. Please upgrade to v1.0.1
+to use the latest version of the support libraries.
+
```

Comment:
What do you think of this statement (assuming FirebaseUI v1.0.1 contains this PR).